scripts/setup-conf.sh

```diff
@@ -39,12 +38,14 @@ timezone='${TIMEZONE}'
 cron.use_background_workers = on
 EOF
 
-# This script will setup necessary replication settings
-
-
+echo "include 'postgis.conf'" >> $CONF
 
+# Create a config for logical replication
 if [[  "${REPLICATION}" =~ [Tt][Rr][Uu][Ee] && "$WAL_LEVEL" == 'logical' ]]; then
-cat >> "$CONF" <<EOF
+  if [[  -f ${ROOT_CONF}/logical_replication.conf ]];then
+    rm $CONF/logical_replication.conf
+  fi
+cat >> ${ROOT_CONF}/logical_replication.conf <<EOF
```

Comment:
I fixed some typo. It was previously `streaming_replication.conf` so the test fails in `logical_replication` scenario